components/photo.js

```diff
@@ -0,0 +1,43 @@
+import React, { PropTypes } from 'react';
+
+const Photo = (props) => (
```

Comment:
changed impl for Photo to stateless, so don't need to extend React.Component anymore